src/test/java/ui/ConfigProperties.java

```diff
@@ -0,0 +1,31 @@
+package ui;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigProperties {
+    protected static Properties properties;
+    protected static FileInputStream fileInputStream;
+
+    static {
```

Comment:
nit: I personally don't like static variables and blocks if there's no good reason for them. It's not quite easy to mock them. 

Would it be better to have a regular bean with a normal constructor, etc?